README.md

```diff
@@ -652,7 +652,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 - :video_camera: [GraphQL tutorial for beginners](https://www.howtographql.com/)
 - :books: [GraphQL Tutorials](https://www.tutorialspoint.com/graphql/index.htm)
-- :video_camera: [Complete App with GraphQl](https://www.youtube.com/playlist?list=PL55RiY5tL51rG1x02Yyj93iypUuHYXcB_)
+- :video*camera: [Complete App with GraphQl](https://www.youtube.com/playlist?list=PL55RiY5tL51rG1x02Yyj93iypUuHYXcB*)
```

Comment:
Remove the change here as it is messing the existing resource.

Reply:
Not sure what happened there but it's been corrected. Thanks!